### PR TITLE
initial db changes for batch scheduler

### PR DESCRIPTION
**Pull request by justinfisk**
multiple ongoing tasks need these changes (i.e. #123 #124 #125) so seems that this should be merged into dev. 

I believe this has no negative effects and that the existing gips workflow will not be impacted. 

It may be appropriate to expand this pull request to include other shared DB changes that will also be needed, so I am mostly submitting this PR for discussion.

### CODE REVIEW

File: gips/inventory/dbinv/api.py [hunk 1]
```diff
@@ -214,7 +214,7 @@ def add_product(**values):
     return p # in case the user needs it
 
 
-def update_or_add_asset(driver, asset, tile, date, sensor, name):
+def update_or_add_asset(driver, asset, tile, date, sensor, name, status):
```
Comment (ags-tolson) [inline gips/inventory/dbinv/api.py]: Suggest adding a default of 'complete' to to the `status` arg for this fn and `update_or_add_product`.  That way you don't have to modify `gips/scripts/archive.py` nor fix this function's unit test(s?).

File: gips/inventory/dbinv/api.py [hunk 2]
```diff
@@ -226,13 +226,14 @@ def update_or_add_asset(driver, asset, tile, date, sensor, name):
...
         'date':   date,
+        'status': models.Status.objects.get(status=status),
     }
     update_vals = {'sensor': sensor, 'name': name}
     (asset, created) = models.Asset.objects.update_or_create(defaults=update_vals, **query_vals)
     return asset # in case the user needs it
 
 
-def update_or_add_product(driver, product, tile, date, sensor, name):
+def update_or_add_product(driver, product, tile, date, sensor, name, status):
```
Comment (ags-tolson) [inline gips/inventory/dbinv/api.py]: just to tag it for git's review tracker thing - Suggest adding a default of 'complete' to to the status arg for this fn

File: gips/inventory/__init__.py [hunk 3]
```diff
@@ -266,6 +266,7 @@ def add_to_collection(date, tile, kind, item):
                 'driver': Repository.name.lower(),
                 'tile__in': spatial.tiles,
                 'date__in': dates,
+                'status'  : dbinv.models.Status.objects.get(status='complete'
```
Comment (ags-tolson) [inline gips/inventory/__init__.py]: This doesn't look like it parses?

Comment (justinfisk) [inline gips/inventory/__init__.py]: sloppy conflict resolution. will fix

**Comment by ircwaves** [comment]
Tom's request makes sense.  Once those changes are in, we can run the tests and merge

**Comment by ags-tolson** [comment]
This isn't merging any time soon; this branch is being repurposed to act as a prototype for the demo we have to do soon.

**Pull request closed without merging**